tendrl/gluster_integration/message/callback.py

```diff
@@ -19,6 +19,7 @@
 RESOURCE_TYPE_BRICK = "brick"
 RESOURCE_TYPE_PEER = "host"
 RESOURCE_TYPE_VOLUME = "volume"
+SYNC_INTERVAL = NS.config.data.get("sync_interval", 10)
```

Comment:
It will give error, NS is not available at this stage

Reply:
callback import is happening before NS create https://github.com/Tendrl/gluster-integration/blob/master/tendrl/gluster_integration/manager/__init__.py#L13.


So we can't use NS  here

Reply:
Thanks for pointing that.